src/main/flight/pid.c

```diff
@@ -180,8 +179,8 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .launchControlAllowTriggerReset = true,
     );
 #ifdef USE_DYN_LPF
-    pidProfile->dterm_lowpass_hz = 120;
-    pidProfile->dterm_lowpass2_hz = 180;
+    pidProfile->dterm_lowpass_hz = 150;
+    pidProfile->dterm_lowpass2_hz = 150;
```

Comment:
I really don't like this. Changing the defaults based on whether a feature is included in the code or not is a bad idea. Particularly since the value takes on different function with/without the feature. So assume that this code can't be included for F3, then those users will have different filtering defaults without knowing why. They might try to use some settings from somebody else's tuning guide (that has the feature) and unknowingly set bad values.  I think we need to separate the settings for normal lowpass and dynamic (min/max).  Then allow the user to select between the filter types explicitly - not implicitly.

Reply:
USE_DYN_LPF is intended for troubleshooting/debug.  Dynamic lowpass is being built in the F3 targets.  I'm trying to be as non-intrusive as practical, so the feature can easily be removed if necessary.  This filter strategy requires reconfiguring the 2nd gyro and dterm filters, but I'm trying to preserve the old configuration in case it's built without dynamic lowpass.

Reply:
> USE_DYN_LPF is intended for troubleshooting/debug.

Ok, in this case this is wrong and has to be changed - we need to be able to build targets without features enabled if they don't fit into flash, or else we won't be able to build them at all.

Please fix this so `USE_DYN_LPF` becomes useable for disabling the feature for release builds, and then let us know so we can review.

Reply:
Agreed with both @etracer65 and @mikeller. Explicit choices and match existing behavior of `USE_` define(s) throughout codebase.

Reply:
It does disable it.  Almost everything is wrapped in a define and maintains the previous filter behavior.